db/migrate/20180928100152_create_one_time_responses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOneTimeResponses < ActiveRecord::Migration[5.0]
+  def change
+    create_table :one_time_responses do |t|
+      t.string :token, null: false 
```

Comment:
Volgens mij moet je hier nog een `unique: true` aan toevoegen. De regel in schema.rb waar die index staat moet namelijk een unique: true hebben, zoals bv:
Die uniqueness van token moet je eigenlijk ook nog in de database zetten, bv zoals we dat bij uuid van responses doen:
```ruby
add_index :responses, :uuid, unique: true
```